tsconfig.json

```diff
@@ -26,9 +26,7 @@
     "allowJs": true,
     "forceConsistentCasingInFileNames": true,
     "esModuleInterop": true,
-    "typeRoots": ["./src/types", "./node_modules/@types"]
+    "typeRoots": ["@types", "./node_modules/@types"]
   },
-  "include": [
-    "src/**/*"
-  ]
+  "include": ["src/**/*"]
```

Comment:
Moved the `src/types` directory to `@types`. I _think_ this is more conventional, but let me know if there's a strong reason to have custom declaration files in the `src` directory.